modules/ROOT/pages/about-flows.adoc

```diff
@@ -42,20 +37,27 @@ out to other services, you might factor out that behavior into other flows.
 
 == Using Subflows
 
-You can also group a series of event processors into subflows. Like a flow, a
-subflow is a scope that groups together a sequence of event processors, but it
-does not have an event source, and it does not have its own error handling.
-Subflows are always called from another flow or subflow through a Flow Ref
-component.
+A subflow is a scope that enables you to group event processors, similar to a flow, but with certain differences and limitations:
```

Comment:
I suggest italics in one situation only: when introducing/defining a "new" term:
```suggestion
A _subflow_ is a scope that enables you to group event processors in a manner similar to that a flow but with certain differences and limitations:
```